objects/rct2/scenario_text/rct2.scenario_text.alpine_adventures.json

```diff
@@ -0,0 +1,71 @@
+{
+ "id": "rct2.scenario_text.alpine_adventures",
+ "authors": [
+  "Chris Sawyer"
+ ],
+ "version": "1.0",
+ "sourceGame": [
+  "rct2"
+ ],
```

Comment:
For all scenarios that have one, I’d like to have the corresponding `originalId` here.

(The DAT examiner may help:) https://goes.rctspace.com/dat-examiner